EIPS/eip-7231.md

```diff
@@ -0,0 +1,175 @@
+---
+eip: 7231
+title: Identity aggregated NFT
+description: The aggregation of web2 & web3 identities to NFTs, authorized by individuals, gives attributes of ownerships, relationships, experiences.
+author: Chloe Gu <chloe@carv.io>, Navid X. (@navid), Victor Yu <victor@carv.io>, Archer H.
+discussions-to: https://ethereum-magicians.org/t/eip-7231-namespaced-storage-layout/14796
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-25
+requires: 165, 721, 1271, 6066
+---
+
+## Abstract
+
+This standard extends [ERC-721](./eip-721.md) by binding individuals' Web2 and Web3 identities to non-fungible tokens (SBTs included). By binding multiple identities, aggregated and composible identity infomation can be verified, resulting in more beneficial onchain scenarios for individuals, such as self-authentication, social overlapping, commercial value generation from user targetting, etc. 
```

Comment:
It'd be nice to include a short _technical_ summary of the proposal in the abstract. You should answer the question of: how does this proposal attach identities to NFTs?

Reply:
add the technical summary  as follow：
By adding a custom schema in the metadata, and updating and verifying the schema hash in the contract, the binding of NFT and identity information is completed.